src/components/Cron/HumanReadableCron.js

```diff
@@ -1,11 +1,8 @@
 import { PropTypes } from '@dhis2/prop-types'
 import { useHumanReadableCron } from '../../hooks/human-readable-cron'
 
-const HumanReadableCron = ({ cronExpression }) => {
-    const humanReadableCron = useHumanReadableCron(cronExpression)
-
-    return humanReadableCron
-}
+const HumanReadableCron = ({ cronExpression }) =>
+    useHumanReadableCron(cronExpression)
```

Comment:
Having a component that only returns the return value of a hook seems a bit over-engineered. I guess this is not really related to the current PR, and there could be valid reasons for implementing it like this.

Reply:
Yeah it is. Reason it's currently like this is because if I'd be using the hook I would have to call it conditionally, and I believe that's a no no for hooks. There's probably a better way to solve it, but that's the reason it's like this for now.